### PR TITLE
Added snippet for PEFT

### PR DESCRIPTION
**Pull request by merveenoyan**
Hey @pacman100, this PR adds snippets to Hub for PEFT. Can you check if the snippet is general enough?

### CODE REVIEW

File: js/src/lib/interfaces/Libraries.ts
Comment (osanseviero) [inline js/src/lib/interfaces/Libraries.ts:416]: At the moment `config` is not populated, you will need to do a PR in moon landing to add the `base_model_name` and `task_type` in `moon-landing` first. You can then check with something like

```python
from huggingface_hub import model_info
model_info("Mursel/falcon-7b-instruct-finetuned")
```

File: js/src/lib/interfaces/Libraries.ts
Comment (merveenoyan) [inline js/src/lib/interfaces/Libraries.ts:22]: I'm aware of this indent, will lint

File: js/src/lib/interfaces/Libraries.ts
Comment (merveenoyan) [reply js/src/lib/interfaces/Libraries.ts:416]: I think I already did in 6640 in moon-landing, sorry I haven't asked for review there @osanseviero

**Comment by merveenoyan** [comment]
Screenshot 2023-06-13 at 18 15 03
@pacman100 @osanseviero ignore the first part, it's irrelevant to this PR (tried in other libraries and doesn't go away for some reason)

File: js/src/lib/interfaces/Libraries.ts
Comment (osanseviero) [inline js/src/lib/interfaces/Libraries.ts:415]:
```suggestion
	const peftBaseModel = model.config?.peft?.base_model_name;
```

**Review by osanseviero — COMMENTED** [review]
Overall looks good! I'll wait for @pacman100 review 🔥 thanks a lot!

**Review by osanseviero — COMMENTED** [review]
We also need to add a row in https://github.com/huggingface/hub-docs/blob/main/docs/hub/models-libraries.md for this

**Review by pacman100 — APPROVED** [review]
Thank you so much @merveenoyan for adding support for showing code snippets on hub with respect to using PEFT models 🤗🔥✨. LGTM!

**Comment by merveenoyan** [comment]
Screenshot 2023-06-14 at 15 33 12

**Comment by merveenoyan** [comment]
@SBrandeis sorry I accidentally merged, didn't see that Omar has asked for a review from you, so I reverted it.

**Pull request merged**
